### ISSUE TITLE
Invite users with no profile name to provide one immediately upon login

### ISSUE THREAD

**Issue by rolandturner** [fossasia/open-event-frontend#7041]
**Is your feature request related to a problem? Please describe.**
A repeated problem during FOSSASIA 2021 was speakers who somehow didn't have their name in their Eventyay profile appearing in BBB with a humorous but unwanted pseudonym ("Diligent Bandicoot"), having to ask how to fix this, relying on someone to tell them to click on https://eventyay.com/account/profile (or worse, giving them step-by-step instructions from login), update and save their details, then reload the BBB page (and therefore have to go through the entry procedure again, which in some cases takes minutes).

**Describe the solution you'd like**
A better approach could be for Eventyay to check immediately upon login whether the user has a valid profile name and, if they don't, take them to a special page (**not** the account profile page) which clearly explains why they've been diverted there, asks only for the name field(s) **required** to make BBB present a sensible name, and provides options to save the change or to skip making a change for now. Either way, Eventyay should then continue to wherever it was heading before the login process was initiated.

**Comment by rolandturner** [comment]
> hi @rolandturner Can I take up this issue

Yes, certainly. I am merely proposing an improvement as someone who served as moderator and support person during an event that used Eventyay. I am neither working to implement it myself, nor planning to, so am very happy for any interested person to pick it up!

**Comment by AnshMishra2001** [comment]
May I take this issue up ?

**Comment by AnshMishra2001** [comment]
@rolandturner

**Comment by rolandturner** [comment]
No objection.

**Comment by raj-ship-it** [comment]
i am working on this issue

**Comment by rolandturner** [comment]
A popup would be a reasonable example of a special page, certainly. The important ideas are:
- Intoducing a person in public with a name that was chosen without their awareness is an astonishingly bad idea, particularly when they're speaking at a conference.
- Dumping them into the existing profile page without explanation would not be helpful.
- The inserted interaction should be stripped to the bare minimum required to resolve the problem.

**Comment by progmatic-99** [comment]
Possible duplicate of #7593

**Comment by rolandturner** [comment]
I'd suggest that #7593 is a dup of this ticket :-) But, sure, it's the same concern, and the newer ticket is more prescriptive.

**Comment by Tanish-Panwar** [comment]
@rolandturner  can I work with this issue

**Comment by progmatic-99** [comment]
@maze-runnar, I think this is solved ??
